sbomtool/README.md

```diff
@@ -76,12 +84,38 @@ Generate both SPDX and CycloneDX SBOMs:
 sbomtool generate --name mypackage --build-dir ./build --out-dir ./sbom --spdx --cyclonedx
 ```
 
+Merge multiple SPDX SBOMs:
+```
+sbomtool merge --output merged.json app1-spdx.json app2-spdx.json lib1-spdx.json
+```
+
+Merge with debug logging:
+```
+sbomtool --log-level debug merge --output final.json *.json
+```
+
 ## Output
 
 The tool generates SBOM files in the specified output directory:
 - `{name}-spdx.json`: SPDX format SBOM
 - `{name}-cyclonedx.json`: CycloneDX format SBOM
 
+## Deduplication Behavior
+
+The merge command uses intelligent deduplication to combine packages from multiple SBOMs:
+
+### CPE-Based Deduplication
+- **Primary Strategy**: Uses Common Platform Enumeration (CPE) as the canonical identifier
```

Comment:
You have already mentioned `CPE` earlier in the README, I think you can just use `CPE` here.